WearTilesKotlin/app/src/debug/AndroidManifest.xml

```diff
@@ -17,20 +17,4 @@
     package="com.example.wear.tiles">
 
     <uses-feature android:name="android.hardware.type.watch" />
-
-    <application>
-
-        <activity
-            android:name=".MainActivity"
```

Comment:
Q: Is it useful to have an Activity in case you want to launch something on a click?

Reply:
yep but let's add it back when we do that (maybe in the next pr + 1)

Reply:
good call out - atm we just have the no-op clicks and it's worth showing how to handle it